src/molecules/OverlayingPlugLabels.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import { getVariable } from '../utils';
+
+import { OverlayingPlugLabel } from '../atoms/OverlayingPlugLabel';
+
+const Footer = styled.footer`
+	order: ${props => props.order};
+	display: flex;
+	justify-content: center;
+	z-index: 1;
+	margin-top: calc(-1/2 * (${getVariable('uiSmallLineHeight')} + 2/8 * ${getVariable('verticalBase')}));
```

Comment:
It is a good idea to add negative margins?

Reply:
This is what we use to move the `<footer>` with labels halfway onto the image.